### DISCUSSION TITLE
Feature Request: quick buttons to find issues Authored by Me, Commented by Me, Subscribed by Me, Upvoted by Me

### DISCUSSION THREAD

**Discussion by HariSekhon** [community/community#19601] [Other Feature Feedback, Questions, & Ideas]
I've raised quite a few discussions and feature requests recently and often find myself searching for things I raised to reference at work in Jira tickets or related discussions or GitHub Issues tickets elsewhere or even just to refer back to myself to check or update with workarounds.

Therefore I propose quick button filters for the following:

- "Authored by me"
- "Commented by me"
- "Subscribed by me"
- "Upvoted by me"

These might be best served with a drop-down button for usability vs UI space.

**Comment by creyD** [comment]
This would probably be a problem for https://github.com/me as the username me actually exists.

**Reply by HariSekhon** [reply]
In that case the Buttons suggested are the way to go as they could expand to the actual user and not literal `me`

**Reply by creyD** [reply]
Yeah that should be possible
